src/Resources/views/CRUD/batch_confirmation.html.twig

```diff
@@ -39,7 +39,10 @@ file that was distributed with this source code.
                 {% endif %}
             </div>
             <div class="box-footer clearfix">
-                <form action="{{ admin.generateUrl('batch', {'filter': admin.filterParameters}) }}" method="POST">
+                {{ form_start(form, {
+                    'method': 'POST',
+                    'action': admin.generateUrl('batch', {'filter': admin.filterParameters})
+                }) }}
                     <input type="hidden" name="confirmation" value="ok">
```

Comment:
Should not we also try to replace these inputs?

Reply:
Same here: https://github.com/sonata-project/SonataAdminBundle/pull/7727/files#r822988289